test/src/service/data_products/filtering_test.py

```diff
@@ -190,9 +191,14 @@ def test_filterset_w_defaults():
         ANALYZER(v4_prefixes ALL == doc.fulltextfield, "text_rs")
         AND
         doc.datefield <= @v5_high
+        LIMIT @skip, @limit
         RETURN doc
 """.strip() + "\n"
     assert bind_vars == {
+        "@view": "my_search_view",
```

Comment:
why var view has an extra @?

Reply:
I'm actually not sure of the reasoning, it's not explained: https://www.arangodb.com/docs/stable/aql/fundamentals-bind-parameters.html#syntax

Reply:
> A special type of bind parameter exists for injecting collection names. This type of bind parameter has a name prefixed with an additional @ symbol, so @@name in the query.

I see. Make sense now!